_posts/developers/apps/inrupt-tutorial/2020-11-06-00_getting-started.md

```diff
@@ -27,7 +27,7 @@ to run the application locally on `localhost:1234`.
 
 > You can also try out the application in CodeSandbox.
 >
-> [![Edit on CodeSandbox]({{site.baseUrl}}/assets/img/play-codesandbox.svg)](https://codesandbox.io/s/musing-bouman-n1ghd)
+> [![Edit on CodeSandbox]({{site.baseUrl}}/assets/img/play-codesandbox.svg)](https://codesandbox.io/s/laughing-voice-7xbmf?hidenavigation=1&module=%2Findex.js&view=editor)
```

Comment:
Thanks for updating the codesandbox!
Noticed that the oidcIssuer in the codesandbox still uses a string value instead of the constant.
https://codesandbox.io/s/laughing-voice-7xbmf?hidenavigation=1&module=/index.js&view=editor&file=/index.js:1078-1126

Reply:
I just realised - we can point CodeSandbox to load the code from the repo! Let me set that up, then it'll always be aligned with the code samples here.

Reply:
Alright, done. The link points to the master branch, so will only work after this PR is merged, but this will be the resulting sandbox: https://codesandbox.io/s/github/solid/solidproject.org/tree/getting-started-bad-practices/_posts/developers/apps/inrupt-tutorial/codesandbox/

Reply:
> Alright, done. The link points to the master branch, so will only work after this PR is merged, but this will be the resulting sandbox: https://codesandbox.io/s/github/solid/solidproject.org/tree/getting-started-bad-practices/_posts/developers/apps/inrupt-tutorial/codesandbox/

@Vinnl When clicking on that link the app opens properly in codesandbox, but with an error screen saying:
> A new context *MUST* be provided storage (we expect 'localStorage').

Reply:
@justinwb Hmm, is that in an overlay on the sandbox, or where do you see that? I can't reproduce in both Firefox and Chromium, unfortunately.

Reply:
> @justinwb Hmm, is that in an overlay on the sandbox, or where do you see that? I can't reproduce in both Firefox and Chromium, unfortunately.

It was an overlay, but after clearing cache i don't seem to be able to reproduce in brave (where it was originally happening), or firefox and safari. Will keep an eye out for it again - but no need to hold things up in meantime 🙂